agenda/network/src/main/java/com/pronoidsoftware/agenda/network/KtorRemoteAgendaDataSource.kt

```diff
@@ -113,18 +128,96 @@ class KtorRemoteAgendaDataSource @Inject constructor(
         )
     }
 
+    // Events
+    override suspend fun createEvent(event: AgendaItem.Event): EmptyResult<DataError.Network> {
+        return httpClient.postMultipart(
+            route = AgendaRoutes.EVENT,
+            body = MultiPartFormDataContent(
+                formData {
+                    append(EVENT_CREATE_REQUEST, Json.encodeToString(event.toCreateEventRequest()))
+                    event.photos.forEachIndexed { index, photo ->
+                        val photoName = "photo$index"
+                        append(
+                            photoName,
+                            URL(photo.url).readBytes(),
```

Comment:
What is the URL here for new events? 🤔 
Is `readBytes()` your own extension or how does that work? Wouldn't you get these photos in form of content Uris from the user which you'd have to read in using `contentResolver`?